channel-jira-server/src/main/java/com/synopsys/integration/alert/channel/jira/server/distribution/JiraServerSearcher.java

```diff
@@ -36,6 +42,11 @@ public JiraServerSearcher(String jiraProjectKey, IssueSearchService issueSearchS
                    .collect(Collectors.toList());
     }
 
+    @Override
+    protected TransitionsResponseModel fetchIssueTransitions(String issueKey) throws IntegrationException {
+        return jiraIssueTransitionRetriever.fetchIssueTransitions(issueKey);
+    }
```

Comment:
This can now be deleted and the `JiraIssueTransitionRetriever` can be passed to the superclass.